editor/HFile.cpp

```diff
@@ -650,7 +650,8 @@ char *IntSpacing(int i) {
 
   i = abs(i);
 
-  for (int n = 1; i >= 10; n++)
+  int n;
+  for (n = 1; i >= 10; n++)
     i /= 10;
 
   ASSERT(n <= 20);
```

Comment:
to be safer, this should be a `while` loop, checking that `i>10` and `n<=20`. If `n>20` after, raise error